test/cutest.h

```diff
@@ -78,9 +76,9 @@
  *       TEST_CHECK(ptr->member2 > 200);
  *   }
  */
-#define TEST_CHECK_(cond,...)  test_check__((cond), __FILE__, __LINE__, __VA_ARGS__)
-#define TEST_CHECK(cond)       test_check__((cond), __FILE__, __LINE__, "%s", #cond)
-
+#define TEST_CHECK_(cond, ...)                                                 \
```

Comment:
This line appears to have a ton of white space and then a \\ character now.

Reply:
This is expected.
```
#define TEST_CHECK_(cond,...)  test_check__((cond), __FILE__, __LINE__, __VA_ARGS__)
```
is 85 characters. clang-format wraps it since we have a column limit of 80. For macros the wrapping is done at column 80 for consistency across all macros (instead of some macros wrapping at column 25, 34, 42 etc. which would arguably look a bit messy).

Reply:
Where it wraps makes sense.  It's all the space leading up to that \\ character that was a surprise.  The tool doesn't aim for consistency in other things, like enum assignments, etc.

Reply:
the tool is not perfect